src/sidebar.js

```diff
@@ -84,11 +84,10 @@ module.exports = {
           type: "category",
           label: "Managing Identities",
```

Comment:
```suggestion
          label: "Managing identities",
```

Reply:
Let's do them all in one go, not in this PR ;)